README.md

```diff
@@ -38,6 +38,18 @@ nvm use
 npm install
 ```
 
+### Development
```

Comment:
improve this section. Add some additional notes about rescript development flow!